content/chainguard/chainguard-images/pricing.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
I don't think pricing should be at the top level of Chainguard Containers. "About" makes sense to me, but open to discussing other options.

Reply:
Fair. The page is really about what customers get with what we call "catalog pricing" and not about actual pricing. What if it was called "About Catalog Pricing"?

Reply:
The file title is fine, but you can change it to `catalog-pricing.md` if that makes more sense, I am talking about what directory it lives in

Reply:
Got it. Moving the page with the next commit.